docs/book/platform-guide/set-up-your-mlops-platform/deploy-zenml/deploy-with-custom-image.md

```diff
@@ -0,0 +1,126 @@
+---
+description: Deploying ZenML with custom Docker images.
+---
+
+In most cases, deploying ZenML with the default `zenmlhub/zenml-server` Docker 
+image should work just fine. However, there are some scenarios when you 
+might need to deploy ZenML with a custom Docker image:
+- You have implemented a custom artifact store for which you want to enable
+[artifact visualizations](../../../user-guide/advanced-guide/visualize-artifacts.md)
+or [step logs TODO LINK](../../../user-guide/advanced-guide/visualize-artifacts.md)
+in your dashboard.
+- You have forked the ZenML repository and want to deploy a ZenML server based
+on your own fork because you made changes to the server / database logic.
+
+{% hint style="warning" %}
+Deploying ZenML with custom Docker images is only possible for Docker or Helm
+deployments.
```

Comment:
Maybe link to the relevant docs for both of these?

Reply:
I have links for both below in the respective sections, you think it's useful to add them here as well?

Reply:
Yes. Links are free + while reading I don't necessarily want to keep reading when maybe I just want to go there first.